apis/management.cattle.io/v3/rke_types.go

```diff
@@ -34,6 +34,29 @@ type PrivateRegistry struct {
 	Password string `yaml:"password" json:"password,omitempty"`
 }
 
+type RKESystemImages struct {
+	// etcd image
+	ETCDImage string `yaml:"etcd_image" json:"etcdImage,omitempty" norman:"default=quay.io/coreos/etcd:latest"`
+	// Alpine image
+	AlpineImage string `yaml:"alpine" json:"alpine,omitempty" norman:"default=alpine"`
+	// rke-nginx-proxy image
+	NginxProxyImage string `yaml:"nginx_proxy" json:"nginxProxy,omitempty" norman:"default=rancher/rke-nginx-proxy"`
+	// rke-cert-deployer image
+	CertDownloaderImage string `yaml:"cert_downloader" json:"certDownloader,omitempty" norman:"default=rancher/rke-cert-deployer"`
+	// rke-service-sidekick image
+	ServiceSidekickImage string `yaml:"service_sidekick_image" json:"serviceSideKickImage,omitempty" norman:"default=rancher/rke-service-sidekick"`
+	// KubeDNS image
+	KubeDNSImage string `yaml:"kubedns_image" json:"kubeDNSImage,omitempty" norman:"default=gcr.io/google_containers/k8s-dns-kube-dns-amd64"`
+	// DNSMasq image
+	DNSmasqImage string `yaml:"dnsmasq_image" json:"dnsmasqImage,omitempty" norman:"default=gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64"`
```

Comment:
Based on @vincent99 comments: 
```
`dnsMasq`, `kubeDnsAutoScaler`.  First letter lowercase, only the first letter of subsequent words capital, no special treatment for acronyms
(and sidekick is one word)
or `dnsmasq` actually, since that's the name of the software
```